src/style.css

```diff
@@ -0,0 +1,24 @@
+* {
+  margin: 0;
```

Comment:
Don't reset the margins using * selector. It's not good for performance, better specify the exact tags